### DISCUSSION TITLE
fatal error 'Outer ring of polygon is clockwise' while it is not!

### DISCUSSION THREAD

**Discussion by esevens** [INSPIRE-MIF/helpdesk-validator#840] [Q&A] [answered]
I tried to validate my transformed dataset with https://inspire.ec.europa.eu/validator/home/index.html
I get the fatal error

```
The dataset has 2 feature(s) with errors for this assertion.
XML document 'stortplaatsen_inspire_tf20220726.gml', EnvironmentalManagementFacility 'stortplaatsen_inspire.0': The feature geometry is not a valid GML geometry. Error detected: Invalid polygon. Outer ring of polygon is clockwise within element EnvironmentalManagementFacility, (gml:id: stortplaatsen_inspire.0 with coordinates: LINESTRING (3121099.547725 3927614.957800,3121067.330449 392...)
XML document 'stortplaatsen_inspire_tf20220726.gml', EnvironmentalManagementFacility 'stortplaatsen_inspire.1': The feature geometry is not a valid GML geometry. Error detected: Invalid polygon. Outer ring of polygon is clockwise within element EnvironmentalManagementFacility, (gml:id: stortplaatsen_inspire.1 with coordinates: LINESTRING (3121244.686226 3927611.959234,3121115.170232 392...)
```

I checked the dataset but the outer ring is counter clock wise.

Tested with a simple dataset, 2 records, simple geometries
in zip the tested gml and the test report
[stortplaatsen_inspire_ccw_tf.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/9507155/stortplaatsen_inspire_ccw_tf.zip)

I am stuck now in the process of making a inspire conform dataset

How to continue?

**Comment by JohannaOtt** [comment]
You might have run into https://github.com/inspire-eu-validation/ets-repository/issues/60

**Reply by esevens** [reply]
Indeed, I posted it a few weeks ago also there, but no response. So I am stuck how to continue?

**Reply by JohannaOtt** [reply]
Why would you want to continue? It is an error in the validator, so you would need to ignore is until it is resolved. Or try to validate the data using a different CRS I assume.

**Comment by fabiovinci** [comment, marked as answer]
Dear @esevens,

as mentioned by JohannaOtt, the error is related to a known bug, you can ignore it for the moment.
